tapchannel/aux_invoice_manager.go

```diff
@@ -121,6 +124,14 @@ func (s *AuxInvoiceManager) handleInvoiceAccept(_ context.Context,
 
 	// No custom record on the HTLC, so we have nothing to do.
 	if len(req.WireCustomRecords) == 0 {
+		// If there's no wire custom records and the invoice is an asset
+		// invoice do not settle the invoice.
```

Comment:
nit: Should we be a bit more specific in what happens here? We cancel all HTLCs of the current set.